core/images.go

```diff
@@ -168,11 +168,15 @@ func (n *OpenBazaarNode) FetchImage(peerID string, imageType string, size string
 // GetBase64Image - fetch the image and return it as base64 encoded string
 func (n *OpenBazaarNode) GetBase64Image(url string) (base64ImageData, filename string, err error) {
 	dial := net.Dial
+	var client *http.Client
 	if n.TorDialer != nil {
 		dial = n.TorDialer.Dial
+		tbTransport := &http.Transport{Dial: dial}
```

Comment:
http.Transport.Dial is deprecated and prefers to use DialContext instead. This is fine for testing though as it only affects the Tor case which we aren't testing with at the moment.

Reply:
is the tordialer using .dial?

Reply:
It's possible and might also be way those resources aren't getting released as expected.

Reply:
Yeah proxy does not offer a DialContext method